packages/main/src/SegmentedButton.ts

```diff
@@ -148,27 +126,8 @@ class SegmentedButton extends UI5Element {
 		});
 
 		this.normalizeSelection();
-	}
-
-	async onAfterRendering() {
-		await this._doLayout();
-	}
-
-	prepareToMeasureItems() {
-		this.style.width = "";
-		this.items.forEach(item => {
-			item.style.width = "";
-		});
-	}
-
-	async measureItemsWidth() {
-		await renderFinished();
-		this.prepareToMeasureItems();
 
-		this.widths = this.items.map(item => {
-			// 1 is added because for width 100.44px the offsetWidth property is 100px and not 101px
-			return item.offsetWidth + 1;
-		});
+		this.style.setProperty("--colNum", `${items.length}`);
```

Comment:
change the name of the css variables with something more meaningful and related to the component